### PR TITLE
Try .Net updates -- HashSet`1.xml refresh for out-of-sync PR 3222

### PR DESCRIPTION
**Pull request by WilliamAntonRohm**
old PR -- https://github.com/dotnet/dotnet-api-docs/pull/3222

already merged samples -- https://github.com/dotnet/samples/pull/1533

### CODE REVIEW

**Comment by WilliamAntonRohm** [comment]
@mairaw -- please review these changes.

File: xml/System.Collections.Generic/HashSet`1.xml [hunk 1]
```diff
@@ -588,7 +590,7 @@
 ## Examples  
  The following example creates and populates a <xref:System.Collections.Generic.HashSet%601> collection, then clears it and releases the memory referenced by the collection.
```
Comment (mairaw) [inline xml/System.Collections.Generic/HashSet`1.xml]: missing the note

File: xml/System.Collections.Generic/HashSet`1.xml [hunk 2]
```diff
@@ -693,7 +695,7 @@
 ## Examples
```
Comment (mairaw) [inline xml/System.Collections.Generic/HashSet`1.xml]: missing the note

File: xml/System.Collections.Generic/HashSet`1.xml [hunk 3]
```diff
@@ -933,7 +935,7 @@
 ## Examples  
  The following example demonstrates how to create, populate, and manipulate two <xref:System.Collections.Generic.HashSet%601> objects. In this example, both the contents of the set and <xref:System.Collections.Generic.HashSet%601.Count%2A> display to the console.
```
Comment (mairaw) [inline xml/System.Collections.Generic/HashSet`1.xml]: missing the note

File: xml/System.Collections.Generic/HashSet`1.xml [hunk 4]
```diff
@@ -1072,7 +1074,7 @@
  The following example creates two <xref:System.Collections.Generic.HashSet%601> collections with overlapping sets of data. The lower range of values is then removed from the larger set using the <xref:System.Collections.Generic.HashSet%601.ExceptWith%2A> method.
```
Comment (mairaw) [inline xml/System.Collections.Generic/HashSet`1.xml]: missing the note

File: xml/System.Collections.Generic/HashSet`1.xml [hunk 5]
```diff
@@ -1317,7 +1319,7 @@
 ## Examples  
  The following example creates two disparate <xref:System.Collections.Generic.HashSet%601> objects and compares them to each other. In this example, `lowNumbers` is both a subset and a proper subset of `allNumbers` until `allNumbers` is modified, using the <xref:System.Collections.Generic.HashSet%601.IntersectWith%2A> method, to contain only values that are present in both sets. Once `allNumbers` and `lowNumbers` are identical, `lowNumbers` is still a subset of `allNumbers` but is no longer a proper subset.
```
Comment (mairaw) [inline xml/System.Collections.Generic/HashSet`1.xml]: missing the note

File: xml/System.Collections.Generic/HashSet`1.xml [hunk 6]
```diff
@@ -1389,7 +1391,7 @@
 ## Examples
```
Comment (mairaw) [inline xml/System.Collections.Generic/HashSet`1.xml]: missing the note - can you please review they're not missing in the rest of the changes as well?

**Comment by WilliamAntonRohm** [comment]
@mairaw -- Maira, I've updated the article per your comments -- please review.  As I recall, you mentioned a way to Resolve all conversations at once?

**Comment by mairaw** [comment]
Some can be automatically resolved when you apply my suggestions via the GitHub UI. Otherwise, you need to resolve them individually.

File: xml/System.Collections.Generic/HashSet`1.xml [hunk 7]
```diff
@@ -238,7 +241,9 @@
 ## Examples  
  The following example shows how to create a <xref:System.Collections.Generic.HashSet%601> collection from an existing set. In this example, two sets are created with even and odd integers, respectively. A third <xref:System.Collections.Generic.HashSet%601> object is then created from the even integer set.  
   
- [!code-csharp[System.Collections.Generic.HashSet_UnionWith#02](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Collections.Generic.HashSet_UnionWith/cs/Program.cs#02)]
+ [!INCLUDE[interactive-note](~/includes/csharp-interactive-note.md)]
```
Comment (mairaw) [inline xml/System.Collections.Generic/HashSet`1.xml]: add the note to the MemberGroup element

File: xml/System.Collections.Generic/HashSet`1.xml [hunk 8]
```diff
@@ -377,8 +382,10 @@
 ## Examples  
  The following example uses a supplied <xref:System.Collections.Generic.IEqualityComparer%601> to allow case-insensitive comparisons on the elements of a  <xref:System.Collections.Generic.HashSet%601> collection of vehicle types.  
   
+ [!INCLUDE[interactive-note](~/includes/csharp-interactive-note.md)]
```
Comment (mairaw) [inline xml/System.Collections.Generic/HashSet`1.xml]: duplicate note

File: xml/System.Collections.Generic/HashSet`1.xml [hunk 9]
```diff
@@ -2162,7 +2195,9 @@
 ## Examples  
  The following example creates and populates a <xref:System.Collections.Generic.HashSet%601> collection, and then clears the collection and releases the memory referenced by it.  
   
- [!code-csharp[System.Collections.Generic.HashSet_Clear#02](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Collections.Generic.HashSet_Clear/cs/Program.cs#02)]
+ [!INCLUDE[interactive-note](~/includes/csharp-interactive-note.md)]
+
+ [!code-csharp-interactive[System.Collections.Generic.HashSet_Clear#02](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Collections.Generic.HashSet_Clear/cs/Program.cs#02)]
```
Comment (mairaw) [inline xml/System.Collections.Generic/HashSet`1.xml]:
```suggestion
 [!code-csharp-interactive[System.Collections.Generic.HashSet_Clear#01](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Collections.Generic.HashSet_Clear/cs/Program.cs#01)]
```

File: xml/System.Collections.Generic/HashSet`1.xml [hunk 10]
```diff
@@ -2162,7 +2195,9 @@
...
  [!code-vb[System.Collections.Generic.HashSet_Clear#02](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.Collections.Generic.HashSet_Clear/vb/Program.vb#02)]
```
Comment (mairaw) [inline xml/System.Collections.Generic/HashSet`1.xml]:
```suggestion
 [!code-vb[System.Collections.Generic.HashSet_Clear#01](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.Collections.Generic.HashSet_Clear/vb/Program.vb#01)]  
```

File: xml/System.Collections.Generic/HashSet`1.xml [hunk 11]
```diff
@@ -238,7 +241,9 @@
 ## Examples  
  The following example shows how to create a <xref:System.Collections.Generic.HashSet%601> collection from an existing set. In this example, two sets are created with even and odd integers, respectively. A third <xref:System.Collections.Generic.HashSet%601> object is then created from the even integer set.  
   
- [!code-csharp[System.Collections.Generic.HashSet_UnionWith#02](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Collections.Generic.HashSet_UnionWith/cs/Program.cs#02)]
+ [!INCLUDE[interactive-note](~/includes/csharp-interactive-note.md)]
+
+ [!code-csharp-interactive[System.Collections.Generic.HashSet_UnionWith#02](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Collections.Generic.HashSet_UnionWith/cs/Program.cs#01)]
```
Comment (mairaw) [inline xml/System.Collections.Generic/HashSet`1.xml]:
```suggestion
 [!code-csharp-interactive[System.Collections.Generic.HashSet_UnionWith#01](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Collections.Generic.HashSet_UnionWith/cs/Program.cs#01)]
```

File: xml/System.Collections.Generic/HashSet`1.xml [hunk 12]
```diff
@@ -238,7 +241,9 @@
...
  [!code-vb[System.Collections.Generic.HashSet_UnionWith#02](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.Collections.Generic.HashSet_UnionWith/vb/Program.vb#02)]
```
Comment (mairaw) [inline xml/System.Collections.Generic/HashSet`1.xml]:
```suggestion
 [!code-vb[System.Collections.Generic.HashSet_UnionWith#01](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.Collections.Generic.HashSet_UnionWith/vb/Program.vb#01)]  
```

File: xml/System.Collections.Generic/HashSet`1.xml [hunk 13]
```diff
@@ -377,8 +382,10 @@
...
  The following example uses a supplied <xref:System.Collections.Generic.IEqualityComparer%601> to allow case-insensitive comparisons on the elements of a  <xref:System.Collections.Generic.HashSet%601> collection of vehicle types.  
   
+ [!INCLUDE[interactive-note](~/includes/csharp-interactive-note.md)]
+
  [!code-cpp[System.Collections.Generic.HashSet_ExceptWith#03](~/samples/snippets/cpp/VS_Snippets_CLR_System/system.Collections.Generic.HashSet_ExceptWith/cpp/source2.cpp#03)]
- [!code-csharp[System.Collections.Generic.HashSet_ExceptWith#03](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Collections.Generic.HashSet_ExceptWith/cs/source2.cs#03)]
+ [!code-csharp-interactive[System.Collections.Generic.HashSet_ExceptWith#03](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Collections.Generic.HashSet_ExceptWith/cs/source2.cs#03)]
  [!code-vb[System.Collections.Generic.HashSet_ExceptWith#03](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.Collections.Generic.HashSet_ExceptWith/vb/source2.vb#03)]  
   
  ]]></format>
```
Comment (mairaw) [inline xml/System.Collections.Generic/HashSet`1.xml]: line 544

Can you add the vb version there too?

```markdown
[!code-vb[System.Collections.Generic.HashSet_UnionWith#03](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.Collections.Generic.HashSet_UnionWith/vb/Program.vb#03)]
```

Comment (WilliamAntonRohm) [reply xml/System.Collections.Generic/HashSet`1.xml]: Done.

File: xml/System.Collections.Generic/HashSet`1.xml (hunk 12 above)
Comment (WilliamAntonRohm) [reply xml/System.Collections.Generic/HashSet`1.xml]: Done.

File: xml/System.Collections.Generic/HashSet`1.xml (hunk 8 above)
Comment (WilliamAntonRohm) [reply xml/System.Collections.Generic/HashSet`1.xml]: Done.

File: xml/System.Collections.Generic/HashSet`1.xml (hunk 7 above)
Comment (WilliamAntonRohm) [reply xml/System.Collections.Generic/HashSet`1.xml]: Done.

**Comment by WilliamAntonRohm** [comment]
@mairaw -- Maira, please review these changes.

**Comment by mairaw** [comment]
Needs a new build

**Comment by mairaw** [comment]
There's only one API not working right now: https://review.docs.microsoft.com/en-us/dotnet/api/system.collections.generic.hashset-1.clear?view=netframework-4.8&branch=pr-en-us-3430#code-try-1

I believe with the fix I just made, we should be able to merge this.

**Pull request merged**